staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -697,6 +705,60 @@ func (s *Scheme) addObservedVersion(version schema.GroupVersion) {
 	s.observedVersions = append(s.observedVersions, version)
 }
 
+func objectAPILifecycle(obj Object) (schema.APILifecycle, bool) {
```

Comment:
Somehow document the return types?   The implementation did what I assumed here, but I had to read it to be sure..

Reply:
done.